android/pytorch_android/src/main/java/com/facebook/pytorch/TensorUtils.java

```diff
@@ -0,0 +1,128 @@
+package com.facebook.pytorch;
+
+import android.graphics.Bitmap;
+import android.graphics.ImageFormat;
+import android.media.Image;
+
+import java.nio.ByteBuffer;
+
+public final class TensorUtils {
+  private static float NORM_MEAN_R = 0.485f;
```

Comment:
it seems wrong to just hard-code some transforms numbers in the code. Are you trying to replicate one of the torchvision transforms?

Reply:
I agree.  How difficult is it to pull this into a separate torchvision android library?

Reply:
We can create it in separate gradle dependency with 'tensor utils' and to put it in submodule or just as a sepate gradle library next to 'pytorch_android'